R/FilteredData.R

```diff
@@ -627,8 +627,12 @@ FilteredData <- R6::R6Class( # nolint
       private$filter_panel_active <- FALSE
       fp_id <- self$get_filter_panel_ui_id()
       shinyjs::disable(paste0(fp_id, "-add"), asis = TRUE)
-      private$cached_states <- self$get_filter_state()
-      self$clear_filter_states()
+      slices <- self$get_filter_state()
+      private$cached_states <- slices
+      slices <- slices_drop(slices)
+      slices <- slices_set(slices, list(disabled = TRUE))
```

Comment:
this task can be achieved by simple loop without introducing new 52 lines in teal_slice and 162 lines to test them. 
```suggestion
      for (i in seq_along(slices)) {
        slices[[i]]$disabled <- TRUE
      }
```

Reply:
I have a feeling we will have a need to set more fields sooner or later.

Reply:
So let's introduce these functions later